neqo-transport/src/connection/mod.rs

```diff
@@ -2000,7 +2000,10 @@ impl Connection {
         // Count how many bytes in this range are non-zero.
         let pn_len = mem::size_of::<PacketNumber>()
             - usize::try_from(unacked_range.leading_zeros() / 8).unwrap();
-        // pn_len can't be zero (unacked_range is > 0)
+        assert!(
+            pn_len > 0,
+            "pn_len can't be zero as unacked_range should be > 0, pn {pn}, largest_acknowledged {largest_acknowledged:?}, tx {tx}"
```

Comment:
Seems like codecov is failing, given that this particular line, i.e. the formatting in the case of a panic, is not covered by tests.

https://app.codecov.io/gh/mozilla/neqo/commit/f4af74c4a9fcbeb1b95092f180ea07bd7f110611

I don't think it makes sense to write tests for each of our assertions. If you think differently, I can add a test for this particular one.